### ISSUE TITLE
Textarea

### ISSUE THREAD

**Issue by MarkHesk** [Neop/mudmap2#40]
Hey,

first of all this mapper is cool. Fits perfectly for mapping mud-zones.
My suggestion:
I would like to have the possibility to add a textarea somewhere. 1-n.
To place information there. Sometimes you dont want to click in a room to read everything. Sometimes a textarea between the rooms is enough to describe a certain mechanic.

Possible?

Thanks.
MaRk

**Comment by Neop** [comment]
Thanks for your suggestion!
You mean text on the map itself? It's on my todo list. I'll try to add it some time next week, but I can't promise since it might require some more complex changes.

**Comment by MarkHesk** [comment]
Yes. Text on the map. For example if you have to describe a quest and want to read it while looking at the map. Not checking the comment of a room
